src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,6 +51,25 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    swerveX = swerveSubsystem.getPose().getX();
+    
+    if (swerveX >= 0 && swerveX <= Units.feetToMeters(4)) {
+      simPitch = Units.degreesToRadians((swerveX * (22/Units.feetToMeters(4))) - 11);
+    }
```

Comment:
Use the chargingStation constants from the FieldConstants class to do your calculation

Reply:
Couldn't use the FieldConstants class before because I had not pulled from develop, but this if now fixed and the FieldConstants value is used.